lib/std/json.zig

```diff
@@ -1232,42 +1232,126 @@ pub const Value = union(enum) {
     Array: Array,
     Object: ObjectMap,
 
+    pub fn jsonStringify(
```

Comment:
The "json" part of this function name is redundant. The full path to this function is:

`std.json.Value.jsonStringify`

but it would be better to be

`std.json.Value.stringify`

Reply:
Also this function appears to be redundant with `std.json.WriteStream().emitJson`

Reply:
> The "json" part of this function name is redundant. The full path to this function is:
> 
> `std.json.Value.jsonStringify`
> 
> but it would be better to be
> 
> `std.json.Value.stringify`

This function is a trait.



> Also this function appears to be redundant with `std.json.WriteStream().emitJson`

Yes: I have a followup PR to make that function use this one.

Reply:
> This function is a trait.
To make things clear you are expecting that you can

```
pub fn foo(val: var) []u8 {
   return val.jsonStringify()
}
```

and it will for any enum, struct, or whatever that has a `jsonStingify` method

Reply:
@justinryanH yes. and if you implement `jsonStringify` on your own struct/union/enum then `std.json.stringify` uses that instead of the "normal" routine for that data type.